root/user/components/UserInlineList.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/license/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {l} from '../../static/scripts/common/i18n';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+
+type Props = {|
+  +editors: EditorT,
+|};
+
+const UserInlineList = ({editors}: Props) => {
+  <p>
+    {editors.size ? (
+      {commaOnList(
+        editors.map((editor) => (
+          <EntityLink entity={editor} />
+        )),
```

Comment:
There is an extra closing parenthesis, as the following line should be the second parameter to call the `map` function. Just move the extra parenthesis to line 28.

Reply:
Sorry, I was wrong here, it was correct. The first `)` closed the mapped anonymous function while the second `)` closes the `map` function.